tests/unittests/testTimeScaling.m

```diff
@@ -31,7 +31,7 @@
 % Test all scenarios
 success = 0;
 fail = 0;
-for i=1:num_scenarios
+for i=12:num_scenarios
```

Comment:
Is there a reason why you skip the first 11 scenarios or is this change a typo?

Reply:
I reverted the change.